thirdparty/virtualenv.py

```diff
@@ -1052,10 +1052,10 @@ def copy_required_modules(dst_prefix, symlink):
             if f is not None:
                 f.close()
             # special-case custom readline.so on OS X, but not for pypy:
-            if modname == 'readline' and sys.platform == 'darwin' and not (
+            if modname == 'readline' and is_darwin and not (
```

Comment:
You should not change anything in virtualenv... this is the official/standard code so it should not be modified .